src/Pipelines/Stages/CreateIndexStage.php

```diff
@@ -47,6 +47,15 @@ public function __invoke($payload)
 
         // Reindex data from the old index to the new, but only if the old index exists (not true on brand new setups)
         if ($this->elasticsearchService->indices()->exists(['index' => $payload->getIndexName()])) {
+            // Temporarily change some index settings to speed up the process
+            $this->elasticsearchService->indices()->putSettings([
```

Comment:
I'm not sure we want to do these things during normal indexing. These settings make sense because no one is using the target index until the re-indexing is complete, but for a normal indexing operation the index is in use all the time.